portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -583,6 +583,25 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getGeneticProfileIds': function () {
 	    return this.genetic_profile_ids;
 	},
+	'getDefaultGeneticProfileId': function () {
```

Comment:
I think this should be renamed to make it clear this is specifically for the heatmap. Outside the heatmap there's no notion of a default genetic profile id.